CMakeLists.txt

```diff
@@ -911,21 +941,24 @@ else()
         )
 endif()        
 
-# install python modules
+# install python modules and system libs
 if (PYTHON_COPY)
-    install(DIRECTORY build_tree/${PYTHON_COPY_LIB_TARGET} 
-            DESTINATION lib
-            PATTERN *.dll EXCLUDE
-            PATTERN *.a EXCLUDE
-            PATTERN *.so EXCLUDE
-            PATTERN */dist-packages EXCLUDE
-            PATTERN *.py EXCLUDE)
+    # install all sub folders of Python
+    foreach(PY_SUB_PATH ${PY_SUB_PATHS})
```

Comment:
I see. The problem was probably misunderstanding meaning of PY_SUB_PATHS, see comment before. Just to make it more clear we should rename TARGET_PREFIX e.g. to PY_BUILD_PREFIX and us it here as parameter of DIRECTORY. instead of 'build_tree'.